packages/lib/src/interactions/SelectionTool.tsx

```diff
@@ -19,18 +21,16 @@ import type { CanvasEvent, CommonInteractionProps, Selection } from './models';
 import { MouseButton } from './models';
 import { boundWorldPointToFOV, getModifierKeyArray } from './utils';
 
-interface SelectionState {
-  selection: Selection | undefined;
-  isCancelled?: boolean;
-}
-
 interface Props extends CommonInteractionProps {
   id?: string;
   transformSelection?: (selection: Selection) => Selection;
-  onSelectionStart?: (selection: Selection) => void;
-  onSelectionChange?: (selection: Selection | undefined) => void;
+  onSelectionStart?: () => void;
```

Comment:
With #1339, `onSelectionChange` is always called after `onSelectionStart`, so it makes less sense to pass the `selection` object to `onSelectionStart`.